### DISCUSSION TITLE
Degradation acceleration according to c-rate

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#3273] [Q&A]
I tried to study the effect of Crate on degradation throng CCCV cycles using the SEI growth model and lithium plating in PyBaMM.
I know that increasing the c-rate applied by CCCV increases the temperature inside the battery, which leads to more degradation and shorter battery cycle life.
However, pybamm simulation results show the opposite. 
a. 3c-rate
![3crate_!](https://github.com/pybamm-team/PyBaMM/assets/113442395/ba692fab-a4a8-4a16-8ff8-bc0fce64cfa4)
b. 6c-rate
![6c-rate1](https://github.com/pybamm-team/PyBaMM/assets/113442395/714808b7-0fc6-4ee7-b276-cd507c7e3568)

According to the plot I posted, the capacity loss due to sei growth and lithium plating is greater at 3c-rate than at 6c-rate.
c. 3c-rate
![3c-rate](https://github.com/pybamm-team/PyBaMM/assets/113442395/4d052596-d3ed-414a-8b5b-3d36ca3c5947)
d.6c-rate
![6c-rate](https://github.com/pybamm-team/PyBaMM/assets/113442395/a6fc3dee-5141-4e83-b7e8-414c8cbed1bc)

Also, if you look at the graph(c and d), the important factors of electrolyte_exchange_current_density(eg.graphite_LGM50_electrolyte_exchange_current_density_Chen2020) are c_e : Electrolyte concentration [mol.m-3].
c_s_surf : Particle concentration [mol.m-3] have constant values regardless of c-rate, is this correct?
Any suggestions on how to solve these problems?

**Comment by brosaplanella** [comment]
Note that higher C-rates mean shorter cycles which can result in less ageing (as the battery is used more aggressively but for shorter times). The concentrations should not be constant, they appear so because you are looking at their initial values, but if you move the slider at the bottom of the plot they should change and no longer be constant.

**Reply by zeecode07** [reply]
I also experienced the same thing where the higher the Crates , the more cycles generated, 
why is that? is there something wrong? I use a simulating-long-experiment notebook
@brosaplanella

**Reply by brosaplanella** [reply]
That depends on the simulation. Check the terminal messages to see why the simulation stopped earlier.

**Comment by Sunnyturicum** [comment]
Hi, I am facing a similar issue. However, I am plotting against throughput capacity to avoid your mentioned problem of shorter cycles with less ageing. This resulted in the following graph:

![image](https://github.com/pybamm-team/PyBaMM/assets/63863005/ba06cb2e-c3a8-4a63-bef6-ca6c767bd39d)


In my understanding, comparing at the same throughput capacity should result in a lower degradation for lower C-rates. However, this is not the case. Furthermore, the degradation seems to be faster at 15°C for lower C-rates. How is this possible?

### Used model
```
model = pybamm.lithium_ion.SPMe(
    {
        "lithium plating": "partially reversible",
        "lithium plating porosity change": "true",  # alias for "SEI porosity change"
    }
)
```
### Lower C-rate experiment
```
cycle_number = 500
exp_lunana = pybamm.Experiment(
    [("Discharge at 0.02C until 2.5 V",  # ageing cycles
    "Charge at 0.08C until 3.65 V",
    "Hold at 3.65 V until C/20",
    "Rest for 4 hours",)] * cycle_number
)
```

### Higher C-rate experiment
```
cycle_number = 500
exp = pybamm.Experiment(
    [("Discharge at 1C until 2.5 V",  # ageing cycles
    "Charge at 0.3C until 3.65 V",
    "Hold at 3.65 V until C/20",
    "Rest for 4 hours",)] * cycle_number
)
```

**Reply by brosaplanella** [reply]
That's very interesting! I think this is more of a research question, I have seen similar behaviour in experimental data but I do not know why it occurs. You might want to plot some SEI and plating variables (e.g. thickness or current density) to observe where does the degradation occur.

**Reply by Sunnyturicum** [reply]
Thank you, that was helpful. Plotting it over time also revealed interesting results.

**Reply by Akila1993** [reply]
Hi @Sunnyturicum could you please, provide me instructions on how you plot the different experiments in the same plot? (i.e. for different C-rates and Temperature values) Can you provide me the code for that?

**Reply by ArnaudOlt** [reply]
https://github.com/ArnaudOlt/dissertation_pybamm

**Comment by ganmad** [comment]
If ageing is dominated by calendar ageing part, such as diffusion limited SEI ageing, the time spent by cell at higher SoC/charged state becomes important.  Note that SEI ageing will happen even at zero current as in calendar ageing case, if cell is left at charged state. At lower C rate, the cell is cycled for longer time, means higher SEI. Your results of higher SEI loss at lower C-rate corroborate with this explanation. You may compare SEI aging on t axis as well. Hope this explanation is useful.

**Reply by Thiyaga2025** [reply]
@ganmad ,
thanks for the insight. one thing i observed in Pybamm with diffusion limited SEI aging. SEI aging is not affected by SOC level but lithium plating do. I plotted SEI overpotential it was the same at high and low SOC windows but Lithium plating overpotential changes wrt SOC interval. your thoughts would be helpful.

**Comment by Naveen-S06** [comment]
Could anyone guide me how to plot a data with respect to cycle number? I'm new to PyBamm and it would be great if anyone helps me in this. I don't see cycle number being an output entity in the default parameters set

**Reply by ArnaudOlt** [reply]
It is in the summary variables. Here is an access example in workbook 5.2:  https://github.com/ArnaudOlt/dissertation_pybamm
